browser/brave_news/brave_news_controller_factory.cc

```diff
@@ -29,6 +29,21 @@ BraveNewsController* BraveNewsControllerFactory::GetForContext(
       GetInstance()->GetServiceForBrowserContext(context, true));
 }
 
+// static
+mojo::PendingRemote<mojom::BraveNewsController>
+BraveNewsControllerFactory::GetRemoteService(content::BrowserContext* context) {
```

Comment:
`GetForContext` should return the pending remote, `GetRemoteService` should go away and `GetControllerForContext` duplicates the existing `GetForContext` so I don't understand why it was added.  I see `// Wire up JS mojom to service` in `BraveNewTabUI` so `GetControllerForContext` can be a private method with a friend class for `BraveNewTabUI` until that is updated

Reply:
sorry, not `GetControllerForContext` because `BraveNewsControllerFactory` -> `BraveNewsServiceFactory`, but a private `GetServiceForContext` that returns the raw pointer is ok for now until `BraveNewTabUI` is updated to use the mojo interface directly